### ISSUE TITLE
Make polls deleteable by users

### ISSUE THREAD

**Issue by d3hof** [sampaiodiego/rocket.chat.app-poll#69]
Right know only admins can delete the polls. As far as I understand that is the case, because the poll message is not created by the user but by the poll bot. Is that right? Would there be a way to allow users to delete the polls they created?

**Comment by SufiSR** [comment]
+1, it would be grand to have this option. Ideally, a poll object stores the user it was created / requested by. This user should then be able to successfully delete the poll whereas other users receive an error message.

**Comment by sampaiodiego** [comment]
just like the user can close a poll you guys you'd like to be able to completely delete the poll message, is that right?

**Comment by d3hof** [comment]
Yes excactly! The user that created the poll should be able to delete it.
